tests/typescript_class/extends_implements.ts

```diff
@@ -0,0 +1,20 @@
+class Class extends AbstractClass implements Interface1, Interface2, Interface3, Interface4 {}
+
+class ExtendsAbstractClassAndImplementsInterfaces extends AbstractClass
+  implements Interface1, Interface2, Interface3, Interface4 {}
+
+class ExtendsAbstractClassAndImplementsInterfaces
+  extends AAAAAAAAAAAAAAbstractClass
+  implements Interface1, Interface2, Interface3, Interface4 {}
+
+class ExtendsAbstractClassAndImplementsInterfaces
+  extends AAAAAAAAAAAAAAbstractClass
+  implements
+    Interface1,
+    Interface2,
+    Interface3,
+    Interface4,
+    Interface5,
+    Interface6,
+    Interface7,
+    Interface8 {}
```

Comment:
Could you add a blank line at the end here.

Reply:
What do you mean by this?

    Interface5,
    Interface6,
    Interface7,
    Interface8*added*
{}
    
    Interface5,
    Interface6,
    Interface7,
    Interface8 {}
*added*

Reply:
@azz please check :)

Reply:
Files should have a blank line at the end of them so the snapshots look right.

Reply:
done :)